modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -29,6 +29,10 @@ export enum TransactionType {
   StakingWithdraw,
   // Handle smart contract calls
   ContractCall,
+  // Wallet owner opt-in to receive this asset in his wallet
+  EnableAsset,
```

Comment:
you can't disable an asset, you can disable a token
```suggestion
  EnableToken,
```

Reply:
I was looking on the blockchain terminology and it is something ambiguous the use. But in general terms they use the concept "Asset" in a broader sense than blockchain, and the concept "token" as a representation of an (external) asset inside the blockchain.
I used "Asset" because Algorand use the term "Asset" (instead of token) in the context of ASAs (Algorand Native Assets). 
But if BitGo terminology is more inclined to "Token", if perhaps is a better name.

Reply:
changed